pkg/defaults/templates.go

```diff
@@ -428,3 +428,152 @@ const ParallelsDiskTemplate = `<?xml version='1.0' encoding='UTF-8'?>
         </Shot>
     </Snapshots>
 </Parallels_disk_image>`
+
+// DefaultEdenTemplate is configuration template for Eden
+const DefaultActivateShTemplate = `# This file must be used with "source bin/activate" *from bash*
```

Comment:
do we need to remove `activate.sh.tmpl` and `activate.csh.tmpl`?

Reply:
oh yes, we should remove the files, they're not needed anymore, thx!